gammapy/irf/edisp/kernel.py

```diff
@@ -77,9 +77,9 @@ def pdf_in_safe_range(self, lo_threshold, hi_threshold):
         Parameters
         ----------
         lo_threshold : `~astropy.units.Quantity`
-            Low reco energy threshold
+            Low reconstructed energy threshold.
```

Comment:
If you change here, you will have to change everywhere. Is it worth it?

Reply:
Most of the time it is actually 'reconstructed' in the docstring. So there is currently inconsistency. 
I would say change to 'reconstructed' where necessary.